app/views/idv/inherited_proofing/no_information.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render(
+      'idv/shared/error',
+      type: :warning,
+      title: t('inherited_proofing.errors.cannot_retrieve.title'),
+      heading: t('inherited_proofing.errors.cannot_retrieve.heading'),
+    ) do %>
+  <p>
+     <%= t('inherited_proofing.errors.cannot_retrieve.info') %>
+  </p>
+
+  <%= link_to t('inherited_proofing.buttons.try_again'), root_url, class: 'usa-button usa-button--big usa-button--wide' %>
```

Comment:
Should the "Try again" be going to a page for the user to try again, rather than the root URL?

Reply:
@aduth Yes, that work(page which does asynchronous api call) is still in progress and this is a placeholder for now.